python_modules/dagster/dagster_tests/core_tests/structured_config_tests/test_structured_config_types.py

```diff
@@ -128,10 +128,10 @@ def a_job():
         )
 
 
-@pytest.mark.skip(reason="not yet supported")
-def test_struct_config_map():
+@pytest.mark.parametrize("DictType", [Dict, Mapping])
+def test_struct_config_map(DictType):
     class AnOpConfig(Config):
-        a_string_to_int_dict: Dict[str, int]
+        a_string_to_int_dict: DictType[str, int]
 
     executed = {}
 
```

Comment:
I think parameterize is overkill here and you should just copy paste since it is such a trivial test